components/datadog/fakeintake/ecsFargate.go

```diff
@@ -105,11 +121,12 @@ func NewECSFargateInstance(e aws.Environment) (*Instance, error) {
 	}, opts...); err != nil {
 		return nil, err
 	}
-
-	if err := e.Ctx.RegisterResourceOutputs(instance, pulumi.Map{
-		"host": alb.LoadBalancer.DnsName(),
-	}); err != nil {
-		return nil, err
+	if os.Getenv("DD_DISABLE_LB") == "" {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** You might want to use LookupEnv instead of GetEnv to make a difference between unset variable and variable set to the empty string (unless the variable has to be explicitly set to a non-empty value)
https://pkg.go.dev/os#LookupEnv